share/qtcreator/sfdk/modules/20-building-mb2.json

```diff
@@ -63,7 +63,7 @@
+            "trDescription": "Perform quality checks.\n\nBy default, test suites operate on the artefacts associated with build tree under CWD. Alternatively, single path to a source tree and/or one or more package files may be specified as <artefact> on command line (build tree is always looked for under CWD).\n\nEach test suite realizes testing on one or more of the four recognized test levels. This is indicated in the output produced with the '--list-suites' option, using the keywords found in the first column of the following table.\n\n| Level          | Tested art. | Executed at   | Default |\n|----------------+-------------+---------------+---------|\n|    static      |             |               |         |\n| 1. ├── source  | Sources     | Build host    | +       |\n| 2. └── package | Packages    | Build host    | +       |\n|    dynamic     |             |               |         |\n| 3. ├── unit    | Build tree  | Build host    | -       |\n| 4. └── system  | Packages    | Target device | -       |\n\n(See also QUALITY ASSURANCE in the domain help.)\n\n-l, --levels [+|-]<level>...\n    Select testing levels. Levels are selected using the keywords 'source', 'package', 'static', 'unit', 'system' and 'dynamic' as described before.\n\n--list-suites\n    List available test suites.\n\n-s, --suites [+|-]<suite>...\n    Select test suites to use. By default all and only the essential suites are selected. See '--list-suites'.\n\nThe '--levels' and '--suites' options may be passed multiple values, separated with comma. If a name is prefixed with '+' or '-', the effect is cumulative and persists until overriden with later name prefixed with the opposite operator."
```

Comment:
In the original code, there were nonbreakable spaces used before the "static" and "dynamic" words to ensure emacs aligns the table content nicely. I prepared a fixup commit for your convenience - it is in the jb47430-fix branch, please squash it with your commit https://github.com/sailfishos/sailfish-qtcreator/compare/jb47430-fix

Reply:
Done